lib/dndmonster.sty

```diff
@@ -282,6 +285,11 @@
       \item [\sensesname]    \l__senses_tl
       \item [\languagesname] \l__languages_tl
       \item [\challengename] \__dnd_cr_xp:N {\l__challenge_tl}
+      \tl_if_empty:NF {\l__proficiency_tl}
+        {
+          {\enit@svlabel{\enit@format{\pbonname}}}
+          \hskip \labelsep \l__proficiency_tl
```

Comment:
This is a bit of a hack, (ab)using `enumitem` internals to style the label as if it were an `\item` without actually being one, since I wasn't able to find a way to switch the `description` to doing inline items in the middle.